src/processTargets/marks/ThatStage.ts

```diff
@@ -16,7 +11,7 @@ export class ThatStage implements MarkStage {
       throw Error("No available that marks");
     }
 
-    return selectionsToTarget(context.thatMark);
+    return context.thatMark;
```

Comment:
`thatMark` is now a target, so we just return it.  Note that in this PR, it's always just an `UntypedTarget`, but now actions have the option to return something richer